### ISSUE TITLE
CoP Rabbits no longer using TP moves. 

### ISSUE THREAD

**Issue by korietsu** [DarkstarProject/darkstar#756]
I know @TeoTwawki  fixed this as per https://github.com/DarkstarProject/darkstar/pull/674 but it looks like this issue is back again. Not sure what's happening this time. The family system, mobs, and skills look to be in place properly. This is as recent as this commit: https://github.com/DarkstarProject/darkstar/commit/92cd7338c0b200c66567eeaa4f8bb48a8f4ad019

**Comment by TeoTwawki** [comment]
Someone recently told me they saw urgantise use wild carrot, so probably a wrong ID elsewhere caused a conflict. When I added the ID in mob family table I know it was unused, but if its in mobskill it wouldn't be the 1st time someone had an ID tehre that didn't exist in mob family yet..I'll check it out soon

**Comment by korietsu** [comment]
Yep, I'm confused too at the moment. Let the rabbits wail on me for a solid half hour and nothing.

**Comment by TeoTwawki** [comment]
Wanna check the Urganites? If they use wild carrot ima be hearing the twilight zone theme till we find out why.

**Comment by korietsu** [comment]
Yeah, I have a couple people looking at it now.

**Comment by korietsu** [comment]
Cue the Twilight zone music. 

http://i.imgur.com/gipjuOl.png

**Comment by mrhappyasthma** [comment]
I'm checking into that now (also curious if I can learn it as BLU from Urganites or whatever they are called).  Ill update with my findings of their TP moves.

**Comment by korietsu** [comment]
Doing some testing has also shown some mob skills being called for the wrong TP moves after implementing Sinuate Rush and Temporal Shift for goldfish in sea. Possible there's a join issue with the abilities list? That's about the only thing I can think of.

**Comment by TeoTwawki** [comment]
@korietsu I think you are onto something there.

**Comment by korietsu** [comment]
Know which core file handles the query for the abilities offhand?

**Comment by TeoTwawki** [comment]
nope haven't had time to look either.

**Comment by xdemolish** [comment]
https://github.com/DarkstarProject/darkstar/blob/master/src/map/ability.cpp#L266

**Comment by korietsu** [comment]
So I ran the query and it returns the correct values for everything at the moment. 

https://drive.google.com/file/d/0B3880ehfJ1jJNDF5clVIYmU5M3M/view?usp=sharing

All this is doing is adding to the confusion for me. The mob skills are properly assigned to the cop rabbit system, still doesn't explain why the urganites are using them nor why Temporal Shift and Sinuate Rush are swapped in my lab. 

Really don't want to do a rebuild right now since im about done implementing the dolphins and aern's tp moves and AI for the mouth open and the bracelets mode. 

Any other ideas as to why the urganites are using the cop rabbit moves?

**Comment by teschnei** [comment]
uragnite was just the wrong ID in the column, it was probably always like that.. haven't looked into rabbits yet

**Comment by TeoTwawki** [comment]
I saw this in my log in IRC when I got up today, reminded me about this. 

There have been several other mobs using incorrect or no skills despite having correct family IDs in both tables and for whatever reason its just specific IDs failing. This is not do to anything custom - was happening on stock DarkStar.

I moved the ID's for the affected rabbits when I redid part of the moblist awhile back, and forgot to make a note here. The ID used in both the family and skill tables matched (and still do), so that can't the cause, but they might work now. Or the entire bug may have been fixed and I just didn't notice the commit doing so. In any case, can someone verify that these rabbits still have a problem or not on latest DarkStar revision? Thank you in advance. If they are still busted I'll debug break the mobskills and try to see wtf is happening.
